src/custom.scss

```diff
@@ -3,12 +3,42 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+$secondary: #ef4345;
+
+.item-box {
```

Comment:
I thought this was replaced by the use of the `PageBlock` component, but you might need to check on that

Reply:
i think you might be right? taking it out doesnt seem to change anything